modules/integration/pages/ledger-quick-start.adoc

```diff
@@ -158,50 +157,158 @@ You are all caught up to block height 109
+
+**Note**: The main net is not live yet; it will be live some time before the publicly announced date, and we'll push the updated image to point to the main net to ensure you can perform testing on the main net beforehand.
+
+=== ICPT-specific Rosetta API details
+
+==== How are accounts generated and verified?
+
+- Generate an ED25519 keypair.
+- The secret key is used for signing transactions.
+- The public key is used for generating a self-authenticating Principal ID. For more information, see: https://sdk.dfinity.org/docs/interface-spec/index.html#_principals.
+- The Principal ID is hashed to generate the account address.
+
+==== How to use the public key to generate its account address?
+
+- Call the `[/construction/derive](https://www.rosetta-api.org/docs/ConstructionApi.html#constructionderive)` endpoint with the hex-encoded 32-byte public key.
+- Call the `pub_key_to_address` function in the JavaScript SDK.
+
+==== How to verify the checksum of an account address?
+
+- After hex decoding, the first 4 bytes is the big-endian CRC32 checksum of the rest of the address.
+- Call `[address_from_hex](https://github.com/dfinity/rosetta-client#working-with-account-addresses)` in the JavaScript SDK. It returns and error if checksum doesn't match.
+- [Here](https://gist.github.com/TerrorJack/d6c79b33e5b5d0f5d52f3a2c5cdacc60) is a Java implementation of address validation logic.
+
+==== What is the `signature_type` / `curve_type` for ED25519?
+
+- `signature_type` is `"ed25519"`
+- `curve_type` is `"edwards25519"`
+
+==== What kinds of transactions may appear in a block, and what do they mean?
+
+- Each block as queried from the `[/block](https://www.rosetta-api.org/docs/BlockApi.html#block)` endpoint contains exactly one transaction. Note that some operations, such as `+burn+`, are not suppoted in Rosetta API calls. 
+
+- Transfer
+    * Operation 0: type `"TRANSACTION"`, subtracts the transfer amount from the source account.
+    * Operation 1: type `"TRANSACTION"`, adds the same transfer amount to the destination account.
+    * Operation 2: type `"FEE"`, subtracts the fee from the source account.
+    - Don't rely on the order above, you can rearrange them in the `/construction/payloads` call, and when parsing transactions in a block, you should check for transaction type and amount sign instead.
+- Mint
+    * Operation 0: type `"MINT"`, adds the minted amount to the destination account.
+- Burn
+    * Operation 0: type `"BURN"`, subtract the burned amount from the source account.
+- `"status"` is always `"COMPLETED"`, failed transactions don't show up in the polled blocks
+
+==== What fee is needed? Can I customize the fee?
+
+- By calling `[/construction/metadata](https://www.rosetta-api.org/docs/ConstructionApi.html#constructionmetadata)`, you can get `suggested_fee`.
+- At the moment, `suggested_fee` is a constant, and the fee specified in a transfer must be equal to it.
+- Fees do not apply to Mint/Burn operations.
```

Comment:
```suggestion
- Fees do not apply to Mint or Burn operations.
```
Not sure why we are use the slssh instead Mint or Burn -- aren't these two separate operations? The slash sort of implies they are the same operation. @bogwar ?

Reply:
I think it was just a grammatical error. I'll modify it.